docs/windows.md

```diff
@@ -0,0 +1,191 @@
+# Windows Support
+
+Starting with the release version [0.10.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.10.0), Cloud Hypervisor has the full Windows guest support.
+
+Requirements
+
+- Host with KVM enabled 
+- [UEFI](uefi.md) capable guest
+- Guest with VirtIO integrated drivers
+
+At the current stage, only UEFI capable Windows images are supported. This implies the presence of the OVMF firmware during the Windows installation an in any subsequent usage. BIOS boot is not supported.
+
+The subsequent sections will tell in detail on how to prepare an appropriate Windows image.
```

Comment:
`The subsequent sections will tell, in detail, how to prepare an appropriate Windows image.`

Reply:
Applied your suggestions, thanks for reading through!